08LessonEight/in-class-project/homework.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+    <title>Blog</title>
+    <link type="text/css" rel="stylesheet" href="css/homework-style.css">
+
+</head>
+
+<body>
+
```

Comment:
try setting a width for the body and giving it margin: 0 auto; to help align the content in the center.